lib/phoenix_pubsub_redis/redis_server.ex

```diff
@@ -67,6 +70,11 @@ defmodule Phoenix.PubSub.RedisServer do
     {:noreply, state}
   end
 
+  def handle_info({:redix_pubsub, redix_pid, :disconnected, %{reason: reason}}, %{redix_pid: redix_pid} = state) do
+    Logger.error "disconnected from pubsub: #{reason}"
+    {:noreply, state}
```

Comment:
Sorry to bikeshed but including the module is weird. And you can always have the module as part of the metadata. My suggestion:

    Logger.error "Phoenix.PubSub disconnected from Redis with reason #{inspect reason} (awaiting reconnection)"